decidim-meetings/app/controllers/decidim/meetings/admin/meetings_controller.rb

```diff
@@ -54,6 +54,22 @@ def destroy
             end
           end
         end
+
+        def organizers
+          respond_to do |format|
```

Comment:
CodeClimate reports this method as not covered by any test, maybe move this logic to a service object so it can be easily unit-tested?